src/tachimawari/imu/node/imu_provider.cpp

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) 2021 Ichiro ITS
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+// THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+#include <memory>
+#include <vector>
+
+#include "tachimawari/imu/node/imu_provider.hpp"
+
+#include "keisan/keisan.hpp"
+#include "tachimawari/control/manager/control_manager.hpp"
+#include "tachimawari/control/packet/protocol_1/model/packet_id.hpp"
+#include "tachimawari/control/controller/module/cm740_address.hpp"
+
+namespace tachimawari
+{
+
+namespace imu
+{
+
+ImuProvider::ImuProvider(std::shared_ptr<tachimawari::control::ControlManager> control_manager)
+: control_manager(control_manager)
+{
+}
+
+keisan::Vector<3> ImuProvider::get_gyro() const
+{
+  float gyro[3] = {0, 0, 0};
```

Comment:
this variable seems unused in this function

Reply:
My bad, I realized this when I revise GitHub action yesterday. But I don't have much more time to fix this, and I forgot to change it until now. Thanks for remembering me

Reply:
done